### ISSUE TITLE
Unnamed entity "O" found while evaluating the model

### ISSUE THREAD

**Issue by sameerkumar16** [RasaHQ/rasa_nlu#814]
**Rasa NLU version** ( `0.10.6`)

**Used backend / pipeline** ( `spacy_sklearn`)

**Operating system** (`Ubuntu 14.04`)

**Python version** (`3.6`)
**Issue**:

I have trained my model with spacy_sklearn pipeline .I was trying to evaluate the model .
In the table for precision ,recall ,f-score and support ,there was an entity with "O" name with a very large 
support score .
      precision    recall  f1-score   support

            O        0.98      1.00      0.99      5994
            competitors       1.00      0.31      0.47        13
            domain       0.93      0.69      0.79        61
            email       1.00      1.00      1.00        10
            feature_info       1.00      0.22      0.36        18
            info       0.50      0.08      0.13        26
            language       0.82      0.67      0.74        21
            location       0.75      0.47      0.58        19
            payment_info       0.00      0.00      0.00         6
            socialchannel       1.00      0.67      0.80        18
            time       1.00      0.15      0.26        20
            vertical       1.00      0.58      0.74        12

            avg / total       0.98      0.98      0.98      6218

 

**Content of configuration file** (if used & relevant):
```json
{
 "pipeline" : "spacy_sklearn",
  "path" : "./projects",
  "mitie_file" : "/home/sameer/Downloads/MITIE-models/english/total_word_feature_extractor.dat",
  "data" : "./data/backup_sameer.json"
}
```

**Comment by gobrewers14** [comment]
related #813

**Comment by twerkmeister** [comment]
That's the score for tokens without an entity label! It gives you an idea how many entities are found where there are none!

**Comment by amn41** [comment]
@twerkmeister is it worth including a statement to that effect in the output? Seems to trip people up quite a bit :)  or else call it `0 (no entity)` or sth

**Comment by tmbo** [comment]
Yes, @twerkmeister it would be great if you could replace `0` with something like `<no entity>` in the tabular output.

**Comment by twerkmeister** [comment]
:+1:

**Comment by amn41** [comment]
also @twerkmeister if you're tweaking this please align the columns in the output

**Comment by twerkmeister** [comment]
@amn41 columns are aligned, but that gets lost when copying to github I assume! The classification report is generated by a function of the metrics module in sklearn

**Comment by tmbo** [comment]
Merged as #820
